xamarin/Xamarin.Forms
# fix for #8564 exception "Delegate to an instance method cannot have null 'this'"

### Description of Change ###
Added null-checks for _titleViewRendererController.TitleViewPresenter and _titleViewRendererController.CommandBar that can be null if there is no corresponding part of the template

### Issues Resolved ### 
- fixes #8564 

### API Changes ###
None

### Platforms Affected ### 

- UWP

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 


Not applicable

### Testing Procedure ###

set _titleViewRendererController.TitleViewPresenter and _titleViewRendererController.CommandBar to null and see that exception goes away

### PR Checklist ###
